nncf/torch/quantization/init_range.py

```diff
@@ -89,12 +90,37 @@ def get_init_config_for_scope_and_group(self, qid: QuantizerId, group: Quantizer
                          "definition!".format(str(qid)))
 
 
+class PTCollectorParams(CollectorParams):
```

Comment:
See, your PTCollectorParams are at the `init_range` level. Perhaps these should be called `PTRangeInitCollectorParams` ; the common class for these should either be at the `nncf/common/quantization/init_range` level (and its naming aligned as well - for instance, `RangeInitCollectorParams` instead of just `CollectorParams`. Also it might be the case that there is no need to generalize this struct on the FW-common level - I leave this decision at your discretion.

Reply:
I see and agree with the statement. I fixed it. Thanks!